tests/functional/jobscript/01-share-lib-bin/flow.cylc

```diff
@@ -0,0 +1,33 @@
+#!jinja2
+[scheduler]
+    [[events]]
+        stall timeout = PT0M
+
+[scheduling]
+    initial cycle point = 1266
+    [[graph]]
+        R1 = install_cold => run_mypythonscript & run_myscript
+
+[runtime]
+    [[install_cold]]
+        script = """
+            cat > "${CYLC_WORKFLOW_SHARE_DIR}/lib/python/mypythonscript.py" <<__HERE__
+            def my_function():
+                print("I can speak Esparanto like a native.")
```

Comment:
```suggestion
                print("I can speak Esperanto like a native.")
```
But spell it like I didn't grow up in Esperantium 🤣

Reply:
Though given the context it's possible the mis-spelling is deliberate...